src/main/java/org/spongepowered/common/service/scheduler/SpongeScheduler.java

```diff
@@ -38,13 +39,14 @@
 import java.util.Optional;
 import java.util.Set;
 import java.util.UUID;
+import java.util.concurrent.TimeUnit;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class SpongeScheduler implements SchedulerService {
 
     private static final SpongeScheduler INSTANCE = new SpongeScheduler();
-    public static final int TICK_DURATION = 50;
+    public static final int TICK_DURATION = (int) TimeUnit.NANOSECONDS.convert(50, TimeUnit.MILLISECONDS);
```

Comment:
I've fixed the internal representation so the API returns in milliseconds.
Since its a static field I've also appended a suffix that specifies the unit of measurement for clarity.